docs/root/configuration/overview/extension.rst

```diff
@@ -0,0 +1,63 @@
+.. _config_overview_v2_extension_configuration:
+
+Extension configuration
+-----------------------
+
+Each configuration resource in Envoy has a type URL in the `typed_config`. This
+type corresponds to a versioned schema. If the type URL uniquely identifies an
+extension capable of interpreting the configuration, then the extension is
+selected regardless of the `name` field. In this case the `name` field becomes
+optional and can be used as an identifier or as an annotation for the
+particular instance of the extension configuration. For example, the following
+filter configuration snippet is permitted:
+
+.. code-block:: yaml
+
+  name: front-http-proxy
+  typed_config:
+    "@type": type.googleapis.com/envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager
```

Comment:
name is still required by the validation, so it needs to be set to something for this? Same below.

Reply:
Yes, it is set to "front-http-proxy" and "front-router" I believe.

Reply:
Oops sorry I'm blind. My bad.